src/payment-flows/wallet-capture.js

```diff
@@ -300,14 +301,16 @@ function initWalletCapture({
           throw new Error(`Instrument has no type`);
         }
 
+        const cmid = clientMetadataID || sessionID;
```

Comment:
`clientMetadataID` is already set to `sessionID` if it doesn't exist, this happens in `checkout components
```suggestion
        const cmid = clientMetadataID || sessionID;
```
```suggestion
```